.github/workflows/build.yml

```diff
@@ -274,13 +274,13 @@ jobs:
           path: |
             llama-${{ env.BRANCH_NAME }}-${{ steps.commit.outputs.short }}-bin-win-${{ matrix.build }}-x64.zip
 
-  windows-latest-cmake-cublas:
+  windows-latest-cmake-cuda:
     runs-on: windows-latest
 
     strategy:
       matrix:
         cuda: ['12.1.0', '11.7.1']
-        build: ['cublas']
+        build: ['cuda']
```

Comment:
I think renaming all the 'CUBLAS' tags to 'CUDA' is problematic because CUDA is not Nvidia only, CuBLAS is. Renaming CuBLAS to CUDA doesn't make sense for other users such as in PR #1087 with ROCm hipBLAS that uses hipBLAS for Cuda code.

Reply:
It's just a tag. It doesn't affect anything. CUDA is more accurate than cuBLAS currently.

Reply:
> It's just a tag. It doesn't affect anything. CUDA is more accurate than cuBLAS currently.

It doesn't make sense because AMD users building for CUDA won't be able to use "LLAMA_CUDA"